static/app/components/events/interfaces/frame/deprecatedLine.tsx

```diff
@@ -61,23 +62,28 @@ type Props = {
    */
   isHoverPreviewed?: boolean;
   isOnlyFrame?: boolean;
+  isSubFrame?: boolean;
```

Comment:
Can you add a comment above this to describe what a "subframe" is? Since it won't be obvious to anyone besides us